tasks/notify.py

```diff
@@ -50,6 +50,7 @@ async def run_async(
         commitid: str,
         current_yaml=None,
         empty_upload=None,
+        checkpoints=None,
```

Comment:
Would this be an instance of `CheckpointLogger`?  I believe all these task arguments need to be serializable in some way (not sure if JSON is used or something else) - so just wanted to make sure this was tested in some sort of E2E fashion.